### ISSUE TITLE
Hi，thanks for your solution about the issue at first!

### ISSUE THREAD

**Issue by zuoyuwei** [onnx/tensorflow-onnx#1791]
Hi，thanks for your solution about the issue at first!

I got the same error during freeze graph, after change the AssignSub to Sub, AssignAdd to Add... it works!
But I find the result different between the freeze graph and the original graph, that is say: 
I give the same input to frozen graph and original graph, but got  the different result after softmax....
could you please help me the different between AssignSub and Sub? or what should I do to got the same results? 
Thanks

_Originally posted by @xwyf05 in https://github.com/onnx/tensorflow-onnx/issues/77#issuecomment-421542717_

**Comment by hwangdeyu** [comment]
Sorry, it is hard to track the closed issue when we have different contexts per user. 

Could you please open a new one and describe the issue and repro code clearly? Thanks..
